apps/main/src/loan/entities/chains.ts

```diff
@@ -1,18 +1,22 @@
-import { getSupportedChains } from '@curvefi/prices-api/chains'
+import { getSupportedChains as getSupportedChainsFromApi } from '@curvefi/prices-api/chains'
 import { getChains } from '@curvefi/prices-api/llamalend'
 import { EmptyValidationSuite } from '@ui-kit/lib'
+import { queryClient } from '@ui-kit/lib/api/query-client'
 import { queryFactory } from '@ui-kit/lib/model'
 
-export const { getQueryOptions: getSupportedChainOptions } = queryFactory({
+const { getQueryOptions: getSupportedChainOptions } = queryFactory({
   queryKey: () => ['prices-api', 'supported-chains'] as const,
-  queryFn: getSupportedChains,
+  queryFn: getSupportedChainsFromApi,
   staleTime: '1d',
   validationSuite: EmptyValidationSuite,
 })
 
-export const { getQueryOptions: getSupportedLendingChainOptions } = queryFactory({
+const { getQueryOptions: getSupportedLendingChainOptions } = queryFactory({
   queryKey: () => ['prices-api', 'supported-lending-chains'] as const,
   queryFn: getChains,
   staleTime: '1d',
   validationSuite: EmptyValidationSuite,
 })
+
+export const getSupportedLendingChains = () => queryClient.fetchQuery(getSupportedLendingChainOptions({}))
```

Comment:
imo `fetchQuery` should be returned from `queryFactory` similarly to how it returns `getQueryData`, but can do that in a diff PR

Reply:
yeah I agree, let's keep separate this is large